test/unit/presenters/document_export_presenter_test.rb

```diff
@@ -128,6 +148,25 @@ class DocumentExportPresenterTest < ActiveSupport::TestCase
     assert_equal "FileAttachment", attachment[:type]
     assert_equal publication_file.attachments.first.attachment_data.as_json.symbolize_keys,
                  attachment[:attachment_data]
+    # TODO - do titles vary per attachment variant?
+    assert_match(/^file\-attachment\-title\-[0-9]$/, attachment[:title])
```

Comment:
I'd like to double check that we don't expect each variant of FileAttachment to have its own title (e.g. "File attachment PDF" vs "File attachment thumbnail").

Currently I believe we just have `:title` on the FileAttachment root, which in this hypothetical scenario would be "File attachment".